src/dune/workspace.ml

```diff
@@ -65,6 +73,8 @@ module Context = struct
         field_o "host" (Dune_lang.Syntax.since syntax (1, 10) >>> string)
       and+ toolchain =
         field_o "toolchain" (Dune_lang.Syntax.since syntax (1, 5) >>> string)
+      and+ fdo_target_exe =
+        field_o "fdo" (Dune_lang.Syntax.since syntax (1, 12) >>> string)
```

Comment:
Note: this will need to be adjusted before merging. It likely needs to be `(2, 0)` if not `(2, 1)`.

Reply:
optimistically changed to (2, 0) and improved tests for fdo along the way.